src/poliastro/twobody/thrust.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+
+from poliastro.twobody.decorators import state_from_vector
+from poliastro.util import norm, circular_velocity
+
+
+def edelbaum_ai(k, a_0, a_f, inc_0, inc_f, f):
+    """Guidance law from the Edelbaum/Kéchichian theory, optimal transfer between circular inclined orbits
+       (a_0, i_0) --> (a_f, i_f), ecc = 0.
+    Parameters
+    ----------
+    k : float
+        Gravitational parameter.
+    a_0 : float
+        Initial semimajor axis.
+    a_f : float
+        Final semimajor axis.
+    inc_0 : float
+        Initial inclination.
+    inc_f : float
+        Final inclination.
+    f : float
+        Magnitude of constant acceleration
+
+    Notes
+    -----
+    Edelbaum theory, reformulated by Kéchichian.
+
+    References
+    ----------
+    * Edelbaum, T. N. "Propulsion Requirements for Controllable
+      Satellites", 1961.
+    * Kéchichian, J. A. "Reformulation of Edelbaum's Low-Thrust
+      Transfer Problem Using Optimal Control Theory", 1997.
+    """
+
+    def _beta_0(V_0, V_f, inc_0, inc_f):
```

Comment:
I would move all these functions outside of the function `edelbaum_ai`. Perhaps we should have a package, and modules inside of it?